src/sentry/ingest/consumer/processors.py

```diff
@@ -149,10 +153,22 @@ def process_event(
         if reprocess_only_stuck_events and not event_processing_store.exists(data):
             return
 
+        is_reprocessed = event_processing_store.exists(data)
+        if is_reprocessed:
+            reprocessed_ids[project_id] + 1
+
+        now = time.time()
         # inc-847: verify mitigation is working
-        if log_inc_847_counter % 1000 == 0 and event_processing_store.exists(data):
-            log_inc_847_counter += 1
-            logger.info(f"inc-847: reprocessing {data}")
+        # TODO: make the 100 an option
+        if len(reprocessed_ids) > 100:
+            logger.info(reprocessed_ids)
+            reprocessed_ids.clear()
+            reprocessed_last_logged_time = now
+
+        if reprocessed_last_logged_time is None or now > reprocessed_last_logged_time + 10:
+            logger.info(reprocessed_ids)
+            reprocessed_ids.clear()
+            reprocessed_last_logged_time = now
```

Comment:
This should be there only if the event exists. We do not want to do this for all the other events in the main consumer

Reply:
also should change to 500? @fpacifici you said it could be more

Reply:
250 it is